test/e2e/utils.go

```diff
@@ -548,3 +549,53 @@ func deletePersistentVolumeClaims(namespace string) {
 		fw.KubeClient.CoreV1().PersistentVolumeClaims(kafkaNamespace).Delete(pvc.Name, &emptyDeleteOptions)
 	}
 }
+
+func verifyCollectorImage(jaegerInstanceName, namespace string, expected bool) {
+	if expected {
+		require.True(t, wasUsingOtelCollector(jaegerInstanceName, namespace))
+	} else {
+		require.False(t, wasUsingOtelCollector(jaegerInstanceName, namespace))
+	}
+}
+
+// Was this Jaeger Instance using the OTEL collector?
+func wasUsingOtelCollector(jaegerInstanceName, namespace string) bool {
+	deployment, err := fw.KubeClient.AppsV1().Deployments(namespace).Get(jaegerInstanceName+"-collector", metav1.GetOptions{})
+	require.NoError(t, err)
+	containers := deployment.Spec.Template.Spec.Containers
+	for _, container := range containers {
+		if container.Name == "jaeger-collector" {
+			logrus.Infof("Test %s is using image %s", t.Name(), container.Image)
+			return strings.Contains(container.Image, "jaeger-opentelemetry-collector")
+		}
+	}
+
+	require.Failf(t, "Did not find a collector image for %s in namespace %s", jaegerInstanceName, namespace)
+	return false
+}
+
+// Returns a collector spec which uses the Open Telementry Collector
+func getOtelCollectorSpec(collectorSpec v1.JaegerCollectorSpec) v1.JaegerCollectorSpec {
+	otelOptions := make(map[string]interface{})
+
+	processors := map[string]string{"queued_retry": ""}
```

Comment:
I would prefer statically defining the `otelOptions` on the line `579`. When it's split to multiple lines it's hard to read.

Reply:
@pavolloffay Can you give me an example of how to do this?

Reply:
Something like this, the other objects can be added analogically. Btw we don't need the queued_retry processor here.
```
	otelCfg := map[string]interface{}{
		"processors": map[string]string{"queued_retry": ""},
	}
```